metaapp/src/androidTest/java/com/sourcepointmeta/metaapp/MetaAppTests.java

```diff
@@ -151,13 +159,14 @@ public void checkNoMessageDisplayWithShowMessageOnce() {
         chooseAction(ACCEPT_ALL);
         Assert.assertTrue(checkFor(CONSENTS_ARE_DISPLAYED));
         navigateBackToListView();
+        Assert.assertTrue(checkForPropertyListScrren());
         tapOnProperty();
         Assert.assertFalse(checkWebViewDisplayedFor(MESSAGE));
         Assert.assertTrue(checkFor(CONSENTS_ARE_DISPLAYED));
     }
 
     @Test
-    public void resetConsentDataAndCheckForMessageWithShowMessageOnce() {
+    public void resetConsentDataAndCheckForMessageWithShowMessageOnce() throws InterruptedException {
```

Comment:
@Suryakantj why do we have to declare `throws InterruptedException`?

Reply:
@andresilveirah We are using wait period till message is visible , for this we are using "countdown" latch. which can throw InterruptedException.